service-app/internal/parser/lp1f_parser/lp1f_validator_test.go

```diff
@@ -19,49 +18,24 @@ func TestValidXML(t *testing.T) {
 }
 
 func TestInvalidXML(t *testing.T) {
-	validator := getValidator(t, "LP1F-invalid.xml")
-	err := validator.Validate()
-	require.Error(t, err, "Expected validation errors due to date ordering but got none")
-
-	messages := validator.(*Validator).baseValidator.GetValidatorErrorMessages()
+	fileName := "LP1F-invalid-dates.xml"
+	validator := getValidator(t, fileName)
 
 	expectedErrMsgs := []string{
 		"(?i)^Page10 Section9 Witness Signature not set",
-		"(?i)^Page10 Section9 Donor Signature not set",
 		"(?i)^Page10 Section9 Witness Full Name not set",
 		"(?i)^Page10 Section9 Witness Address not valid",
-		"(?i)^Page10 Section9 Donor signature not set or invalid",
-		"(?i)^Page12\\[2\\] Section11 Witness Signature not set",
-		"(?i)^no valid applicant signature/dates found",
```

Comment:
Are we able to keep these error in? I assume we'd want to test every possible error here to get maximum coverage?